### ISSUE TITLE
Pretty Print Isn't Working

### ISSUE THREAD

**Issue by amab8901** [meilisearch/meilisearch#2783]
**Describe the bug**
I write the following in my terminal:
```
curl   -X POST 'http://localhost:7700/indexes/meteorites/search'   -H 'Content-Type: application/json'   --data-binary '{ "filter": "mass < 200" }'
```
I get the following output:
```
{"hits":[{"name":"Aachen","id":"1","nametype":"Valid","recclass":"L5","mass":21,"fall":"Fell","_geo":{"lat":50.775,"lng":6.08333}},{"name":"Al Rais","id":"446","nametype":"Valid","recclass":"CR2-an","mass":160,"fall":"Fell","_geo":{"lat":24.41667,"lng":39.51667}},{"name":"Asco","id":"2345","nametype":"Valid","recclass":"H6","mass":41,"fall":"Fell","_geo":{"lat":42.45,"lng":9.03333}},{"name":"Atemajac","id":"4884","nametype":"Valid","recclass":"L6","mass":94.2,"fall":"Fell","_geo":{"lat":20.06667,"lng":-103.66667}},{"name":"Avilez","id":"4907","nametype":"Valid","recclass":"H","mass":146,"fall":"Fell","_geo":{"lat":25.0,"lng":-103.5}},{"name":"Awere","id":"4910","nametype":"Valid","recclass":"L4","mass":134,"fall":"Fell","_geo":{"lat":2.71667,"lng":32.83333}},{"name":"Banswal","id":"4937","nametype":"Valid","recclass":"L5","mass":14,"fall":"Fell","_geo":{"lat":30.4,"lng":78.2}},{"name":"Barnaul","id":"4947","nametype":"Valid","recclass":"H5","mass":23.2,"fall":"Fell","_geo":{"lat":52.73333,"lng":84.08333}},{"name":"Barntrup","id":"4948","nametype":"Valid","recclass":"LL4","mass":17,"fall":"Fell","_geo":{"lat":52.0,"lng":9.1}},{"name":"Bethlehem","id":"5032","nametype":"Valid","recclass":"H","mass":13.9,"fall":"Fell","_geo":{"lat":42.53333,"lng":-73.83333}},{"name":"Bhagur","id":"5037","nametype":"Valid","recclass":"L6","mass":18,"fall":"Fell","_geo":{"lat":20.88333,"lng":74.83333}},{"name":"Bherai","id":"5039","nametype":"Valid","recclass":"L6","mass":100,"fall":"Fell","_geo":{"lat":20.83333,"lng":71.46667}},{"name":"Bloomington","id":"5076","nametype":"Valid","recclass":"LL6","mass":67.8,"fall":"Fell","_geo":{"lat":40.48,"lng":-89.00417}},{"name":"Bocas","id":"5093","nametype":"Valid","recclass":"L6","mass":56,"fall":"Fell","_geo":{"lat":23.0,"lng":-102.0}},{"name":"Boumdeid (2003)","id":"57168","nametype":"Valid","recclass":"L6","mass":190,"fall":"Fell","_geo":{"lat":17.71067,"lng":-11.3715}},{"name":"Caratash","id":"5265","nametype":"Valid","recclass":"LL6","mass":8,"fall":"Fell","_geo":{"lat":38.5,"lng":27.0}},{"name":"Castine","id":"5293","nametype":"Valid","recclass":"L6","mass":94,"fall":"Fell","_geo":{"lat":44.38333,"lng":-68.75}},{"name":"Centerville","id":"5307","nametype":"Valid","recclass":"H5","mass":45.6,"fall":"Fell","_geo":{"lat":43.2,"lng":-96.91667}},{"name":"Chail","id":"5314","nametype":"Valid","recclass":"H6","mass":0.5,"fall":"Fell","_geo":{"lat":25.36667,"lng":81.66667}},{"name":"Chetrinahatti","id":"5344","nametype":"Valid","recclass":"Stone-uncl","mass":72,"fall":"Fell","_geo":{"lat":14.5,"lng":76.5}}],"estimatedTotalHits":114,"query":"","limit":20,"offset":0,"processingTimeMs":0}
```
But I want the output to be pretty-printed instead, like below:
```
{
  "hits":[
    {
      "name": "Aachen",
      "mass": 21
    },
    {
      "name": "Emmaville",
      "mass": 127
    },
    …
  ],
  "estimatedTotalHits": 114,
  "query": "",
  "limit": 20,
  "offset": 0,
  "processingTimeMs": 0
}

```
The pretty-printed version above is taken from this instruction (https://docs.meilisearch.com/learn/getting_started/filtering_and_sorting.html#filtering), according to which the pretty-print is expected to be in the output, yet it's not happening on my side. 

**Screenshots**
![image](https://user-images.githubusercontent.com/83634595/190991264-f5bc3e97-dc11-47c3-94f3-0d47b21f24d0.png)

**Meilisearch version:**
meilisearch-http 0.28.1

**Additional context**
OS: Xfce Garuda, which is a flavor of Garuda Linux, which is a flavor of Archlinux, which is a flavor of Linux, which is a flavor of operating system. 
CPU: Intel

**Comment by ahtinurme** [comment]
@curquiza curious question is that how do you expect the pretty print to work in bash as this is something that can be done on the client side (you), not on the server side (Meilisearch)?

**Comment by amab8901** [comment]
> @amab8901 curious question is that how do you expect the pretty print to work in bash as this is something that can be done on the client side (you), not on the server side (Meilisearch)?

I don't know how it's supposed to work in Bash. I haven't figured out how to do pretty-print in bash. I looked it up and apparently I'm supposed to use the `-i` tag after the `curl` to get pretty print, but even then the pretty print doesn't come. 

I'm copying+pasting the command proposed in the documentation, and it gives a non-pretty print unlike the pretty print that the docs suggest should be displayed. If the input commands suggested by the documentation fail to produce pretty print due to being misformulated, then maybe the documentation should be updated so that copy+pasting the proposed code will result in pretty print

**Comment by amab8901** [comment]
oh, I figured out how to do it. Here's how.

Here is how I typed previously, and it failed to pretty-print the JSON string:
`curl   -X POST 'http://localhost:7700/indexes/meteorites/search'   -H 'Content-Type: application/json'   --data-binary '{ "filter": "mass < 200" }' `

And here's how I'm typing now, and it successfully pretty-prints the JSON string:
`curl   -X POST 'http://localhost:7700/indexes/meteorites/search'   -H 'Content-Type: application/json'   --data-binary '{ "filter": "mass < 200" }' | jq`

The difference lies in the ` | jq` that I appended at the end.
